### PR TITLE
Feature/model group model

### PR DESCRIPTION
**Pull request by EnceladusIApetus**
- all basic functions work functionally.
- the model can now connect to the database.
- the class is already implemented the process status description format.
- required field checking system is implemented to the class.

### CODE REVIEW

File: MoveAlarm-Server/src/main/java/movealarm/kmitl/net/ModelCollection.java [hunk 1]
```diff
@@ -29,6 +29,9 @@ public static ModelCollection getInstance()
         } catch (SQLException e) {
             System.out.println("An error has occurred in ModelCollection.find()");
             System.out.println(e);
+        } catch (Exception e) {
+            System.out.println("An error has occurred in ModelCollection.find()");
+            System.out.println(e);
```
Comment (oat9002) [inline MoveAlarm-Server/src/main/java/movealarm/kmitl/net/ModelCollection.java]: Did it duplicated?

File: MoveAlarm-Server/src/main/java/movealarm/kmitl/net/User.java [hunk 2]
```diff
@@ -84,10 +89,10 @@ public static User find(int id)
 
     public static User[] all()
     {
-        ArrayList<HashMap<String, Object>> temp = modelCollection.all(tableName);
+        ArrayList<HashMap<String, Object>> temp = modelCollection.all("user");
```
Comment (oat9002) [inline MoveAlarm-Server/src/main/java/movealarm/kmitl/net/User.java:92]: use this.tableName instead?

File: MoveAlarm-Server/src/main/java/movealarm/kmitl/net/ModelCollection.java (hunk 1 above)
Comment (EnceladusIApetus) [inline MoveAlarm-Server/src/main/java/movealarm/kmitl/net/ModelCollection.java]: duplicated code was deleted.

File: MoveAlarm-Server/src/main/java/movealarm/kmitl/net/User.java (hunk 2 above)
Comment (EnceladusIApetus) [inline MoveAlarm-Server/src/main/java/movealarm/kmitl/net/User.java:92]: cannot use tableName directly cause of call variable from a static method.

**Pull request merged**